packages/extension/src/tree.view.ts

```diff
@@ -384,7 +384,7 @@ class SnippetItem extends Item implements ContextMenu {
         basePath,
         {
           command: 'marquee.snippet.insert',
-          title: 'Insert Snippet',
+          title: 'Insert Clipboard',
```

Comment:
Let's stay consistent everywhere and call it `Clipboard Item` to avoid confusion with the system clipboard
```suggestion
          title: 'Insert Clipboard Item',
```